packages/libs/deer/src/error.rs

```diff
@@ -1,15 +1,196 @@
+//! # Error Structure
+//!
+//! Every error emitted by `deer` (and potentially other tooling) follows the same schema when
+//! serialized:
+//!
+//! ```json5
+//! {
+//!     "ns": "deer", // namespace of the error
```

Comment:
I'd prefer `namespace` instead of `ns` here

Reply:
fixed in [29cc757](https://github.com/hashintel/hash/pull/1286/commits/29cc7571e8b74c4178e7a4bf50c2e1093d389c6b)